src/Log.ts

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright 2024 Ceeblue B.V.
+ * This file is part of https://github.com/CeeblueTV/web-utils which is released under GNU Affero General Public License.
+ * See file LICENSE or go to https://spdx.org/licenses/AGPL-3.0-or-later.html for full license details.
+ */
+
+import * as Util from './Util';
+
+let _logging = 0;
```

Comment:
What does it do?

Reply:
yes, I was searching a "light" way (not expensive for the CPU!) to raise an assertion if we forget to call error/warn/info/debug, I mean if the developer write by mistake just: `log("my zombie log")`
Each 10 seconds (not CPU expensive) we check that all log built has been used/consumed. Otherwise we alert the developer by an assertion in the code.

Reply:
To be honest, I have hesitated to create an automatic Promise when you call `log("my log")` to raise it automatically in an async way if you forget to call manually error/warn/info/debug. But I think that it's too expensive to create a Promise for each call to `log` I prefer keep it `light` and advise the bad usage to the dev to fix it by required an explicit manual call.